app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectCreationActivity.kt

```diff
@@ -74,8 +76,28 @@ class Form : AppCompatActivity() {
         findViewById<TextView>(R.id.title_form).requestFocus()
         findViewById<Button>(R.id.form_add_subtitle).setOnClickListener(::onClickSubtitleButton)
         findViewById<Button>(R.id.form_button_sub).setOnClickListener(::submit)
+
+        if (projectToEdit != null) {
+            setInitialValues(projectToEdit!!)
+        }
```

Comment:
A ?.let instead of !! could also be used (but it is the same).
```suggestion
        projectToEdit?.let {
            setInitialValues(projectToEdit)
        }
```